tests/collections/db.test.ts

```diff
@@ -84,14 +89,20 @@ for (const testClient in testClients) {
       });
     });
 
-    describe('dropDatabase', () => {
+    describe('dropDatabase', function(this: Mocha.Suite) {
+      const suite = this;
       after(async () => {
-        const keyspaceName = parseUri(process.env.JSON_API_URI).keyspaceName;
+        if(isAstra){
+          return;     
```

Comment:
It is generally better to do `this.skip(); return;` rather than just doing an early `return`, because that tells Mocha that we skipped this `after` rather than executing it successfully.

Reply:
It works inside the `it` block, but not inside the `after` block, so I had to just `return;`
`this.skip(); return;` => `TypeError: this.skip is not a function`
`suite.ctx.skip();` => `Error: `this.skip` forbidden`

Reply:
That's because this `after()` uses an arrow function. Should work if you do `after(function() { if (isAstra) { return this.skip(); } })`

Reply:
My bad, I missed it, I changed it in the previous line in the `describe` and missed here. But I',m still getting `Error: `this.skip` forbidden` let me debug this separately and fix.

But I'm just curious why we need to create a database in the `after` step in this test.

Reply:
Not making this change, since `this.skip(); ` fails with forbidden error and also returning early from the `after` block  is not reported back to mocha as a successful complete and still relies on the return type of `it`.